launcher/common/args.py

```diff
@@ -0,0 +1,72 @@
+# ------------------------------------------------------------------------------
```

Comment:
It's might be good to merge /utils and /common to a single location which provides such short and specific functions used through the framework.